packages/server/pathfinding.ts

```diff
@@ -71,8 +71,14 @@ function gridPathFind(start: TilePos, b: TilePos, m: GameMap) {
             .filter(e => m.tiles[explode(e)] === 0);
 
         for (let next of options) {
+            // detect if moving diagonally
+            const stepCost =
+                (next.x === current.x || next.y === current.y)
+                ? 1
+                : 1.41421356237;
```

Comment:
You can use `Math.SQRT2` if you don't want to compute it every time, but if you don't want a magic number at the same time

Reply:
Nice, didn't know that was in the constants